src/interactive/repl.ts

```diff
@@ -389,12 +389,10 @@ async function evaluate(editor: vscode.TextEditor, range: vscode.Range, text: st
     if (resultType !== 'REPL') {
         if (result.stackframe) {
             results.setStackTrace(r, result.all, result.stackframe)
-            // no need to show details here because the inline error display adds them anyways:
-            r.setContent(results.resultContent(' ' + result.inline + ' ', ''))
         } else {
             results.clearStackTrace()
-            r.setContent(results.resultContent(' ' + result.inline + ' ', result.all))
```

Comment:
@pfitzseb why reverting this ? Removing the duplication looked good to me.

Reply:
Because the stacktrace doesn't stay attached to the result forever (most importantly, only as long as nothing else has been evaluated).

Reply:
makes sense. Related, do you think it makes sense to keep the previous stackframe after the next evaluation ?
I would like to keep them until I hit another stackframe error (and iirc this is what Juno behaves like)

Reply:
Hm, ok. Not sure what I prefer, honestly.

Reply:
Let's see how we feel on this.

Reply:
`e6066b6`